e2e-tests/emb-json/src/main/java/com/foo/rest/emb/json/ocvn/ReleaseJsonToObject.java

```diff
@@ -0,0 +1,33 @@
+package com.foo.rest.emb.json.ocvn;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+
+/**
+ * This code is taken from Open Contracting Vietnam (OCVN)
+ * G: https://github.com/devgateway/ocvn
+ * L: MIT
+ * P: src/main/java/org/devgateway/ocds/persistence/mongo/spring/json2object/ReleaseJsonToObject.java
+ */
+public class ReleaseJsonToObject  {
+    private Release release;
+
+    protected final ObjectMapper mapper;
+    protected final String jsonObject;
+
+    public ReleaseJsonToObject(final String jsonObject, ObjectMapper mapper) {
+        this.mapper = mapper;
+        this.jsonObject = jsonObject;
+    }
+
+
+    public Release toObject() throws IOException {
+        if (release == null) {
+            // Transform JSON String to a Release Object
+            release = this.mapper.readValue(this.jsonObject, Release.class);
```

Comment:
didn't we already handled this case? if so, i guess we could remove OCVN from here

Reply:
For now, this is the only case study which is working for this case. Other tests are disabled for the similar case. That's why left it.